### ISSUE TITLE
Caused by: java.lang.NumberFormatException: Invalid int: "48.0dip"

### ISSUE THREAD

**Issue by y11t** [DreaminginCodeZH/MaterialProgressBar#7]
My app crashed on one of my users. 

I was using  compile 'me.zhanghai.android.materialprogressbar:library:1.1.3'
Their version and device: Android 4.4 - Galaxy A3 (a33g) 

Error:

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mts.quicklhelper/com.mts.quicklhelper.MainActivity}: android.view.InflateException: Binary XML file line #29: Error inflating class me.zhanghai.android.materialprogressbar.MaterialProgressBar
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2327)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2378)
    at android.app.ActivityThread.access$800(ActivityThread.java:155)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1244)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:136)
    at android.app.ActivityThread.main(ActivityThread.java:5433)
    at java.lang.reflect.Method.invokeNative(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:515)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1268)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1084)
    at dalvik.system.NativeStart.main(Native Method)
Caused by: android.view.InflateException: Binary XML file line #29: Error inflating class me.zhanghai.android.materialprogressbar.MaterialProgressBar
    at android.view.LayoutInflater.createView(LayoutInflater.java:627)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:703)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:762)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:771)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:771)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:499)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:398)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:354)
    at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:255)
    at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:109)
    at com.mts.quicklhelper.MainActivity.onCreate(MainActivity.java:87)
    at android.app.Activity.performCreate(Activity.java:5301)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2291)
    ... 11 more
Caused by: java.lang.reflect.InvocationTargetException
    at java.lang.reflect.Constructor.constructNative(Native Method)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
    at android.view.LayoutInflater.createView(LayoutInflater.java:601)
    ... 24 more
Caused by: java.lang.NumberFormatException: Invalid int: "48.0dip"
    at java.lang.Integer.invalidInt(Integer.java:137)
    at java.lang.Integer.parse(Integer.java:374)
    at java.lang.Integer.parseInt(Integer.java:365)
    at com.android.internal.util.XmlUtils.convertValueToInt(XmlUtils.java:122)
    at android.content.res.TypedArray.getInt(TypedArray.java:267)
    at me.zhanghai.android.materialprogressbar.MaterialProgressBar.init(MaterialProgressBar.java:80)
    at me.zhanghai.android.materialprogressbar.MaterialProgressBar.<init>(MaterialProgressBar.java:45)
    ... 27 more

I am using it like this:

<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:ads="http://schemas.android.com/tools"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/coordinatorLayout"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:background="@color/colorBackground">

```
<android.support.design.widget.AppBarLayout
    android:id="@+id/view"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar">

    <android.support.v7.widget.Toolbar
        android:id="@+id/tabanim_toolbar"
        android:layout_width="match_parent"
        android:layout_height="?attr/actionBarSize"
        android:background="?attr/colorPrimary"
        app:layout_scrollFlags="scroll|enterAlways"
        app:popupTheme="@style/ThemeOverlay.AppCompat.Light">

        <Spinner
            android:id="@+id/navSpinner"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:gravity="end" />

        <me.zhanghai.android.materialprogressbar.MaterialProgressBar
            android:id="@+id/progressBar"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_gravity="end"
            android:indeterminate="true"
            android:padding="10dp"
            android:tint="#ffffff"
            android:visibility="gone"
            app:mpb_progressStyle="circular" />

    </android.support.v7.widget.Toolbar>

    <android.support.design.widget.TabLayout
        android:id="@+id/tabanim_tabs"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:tabIndicatorColor="@android:color/white" />

    <View
        android:layout_width="match_parent"
        android:layout_height="4dp"
        android:background="@drawable/shadow" />
</android.support.design.widget.AppBarLayout>

<android.support.v4.view.ViewPager
    android:id="@+id/tabanim_viewpager"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:layout_above="@+id/adView"
    android:layout_below="@+id/view"
    app:layout_behavior="@string/appbar_scrolling_view_behavior" />

<com.google.android.gms.ads.AdView
    xmlns:ads="http://schemas.android.com/apk/res-auto"
    android:id="@+id/adView"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:layout_alignParentBottom="true"
    android:layout_centerHorizontal="true"
    ads:adSize="BANNER"
    ads:adUnitId="@string/banner_ad_unit_id"/>
```

</RelativeLayout>

**Comment by zhanghai** [comment]
Possible duplicate of #6.

**Comment by zhanghai** [comment]
Released as in #6.
